### PR TITLE
TypeScript Definition File 

### PR DESCRIPTION
**Pull request by recurrence**
Here's a basic typescript definition file (as requested in #26 ) :)

### CODE REVIEW

File: src/index.d.ts [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+       * Override the inline-styles of the root element.
+       */
+      style?: any;
+      /**
+       * The value of the text field.
+       */
+      value?: string;
+  }
+
+  export default function SearchBar(props: SearchBarProps): JSX.Element;
```
Comment (leMaik) [inline src/index.d.ts]: Shouldn't this inherit from `ReactElement` or something like that? :thinking:

Comment (recurrence) [reply src/index.d.ts]: Sure, I can set the response to ReactElement

Comment (recurrence) [reply src/index.d.ts]: Updated and squashed!

**Comment by leMaik** [comment]
Seems legit. ¯\_(ツ)_/¯

**Pull request merged**